recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
-{% set version = "1.9.5" %}
+{% set version = "1.10.0a2" %}
 
 package:
   name: holoviews
   version: {{ version }}
 
 source:
-  fn: holoviews-{{ version }}.tar.gz
-  url: https://github.com/ioam/holoviews/archive/v{{version}}.tar.gz
-  sha256: 89bdf9708ab58b43d4c690846867be5559a9cb4b7c215611db646ed3d54c4169
+  git_url: https://github.com/ioam/holoviews.git
+  git_rev: v{{version}}
+  git_depth: 60
```

Comment:
Appears that if we specify `--branch` to `git clone`, this should fix the issue. Added PR ( https://github.com/conda/conda-build/pull/2797 ) to fix this going forward.